buildSDK.sh

```diff
@@ -1,47 +1,9 @@
 #!/bin/sh
 #Copyright (c) Microsoft. All rights reserved.
 #Licensed under the MIT license. See LICENSE file in the project root for full license information.
-if [ ! -d ~/azure-iot-sdk-c ];
-then
-  git clone https://github.com/Azure/azure-iot-sdk-c.git ~/azure-iot-sdk-c && cd ~/azure-iot-sdk-c && git checkout 76906dc;
-fi
-
-cd ~/azure-iot-sdk-c/uamqp
-if ! [ "$(ls -A .)" ];
-then
-  git clone https://github.com/Azure/azure-uamqp-c.git . && git checkout 5bf09d3;
-fi
-
-cd ~/azure-iot-sdk-c/umqtt
-if ! [ "$(ls -A .)" ];
-then
-  git clone https://github.com/Azure/azure-umqtt-c.git . && git checkout 51da812;
-fi
-
-cd ~/azure-iot-sdk-c/parson
-if ! [ "$(ls -A .)" ];
-then
-  git clone https://github.com/kgabis/parson.git . && git checkout c22be79;
-fi
-
-cd ~/azure-iot-sdk-c/c-utility
-if ! [ "$(ls -A .)" ];
-then
-  git clone https://github.com/Azure/azure-c-shared-utility.git . && git checkout 9073d21;
-fi
-
-cd ~/azure-iot-sdk-c/uamqp/c-utility
-if ! [ "$(ls -A .)" ];
-then
-git clone https://github.com/Azure/azure-c-shared-utility.git . && git checkout b0b5b1b;
-fi
-
-cd ~/azure-iot-sdk-c/umqtt/c-utility
-if ! [ "$(ls -A .)" ];
-then
-  git clone https://github.com/Azure/azure-c-shared-utility.git . && git checkout b0b5b1b;
-fi
 
+mkdir ~/azure-iot-sdk-c
+unzip azure-iot-sdk-c-release-2017-11-03.zip -d ~
```

Comment:
where does the `azure-iot-sdk-c-release-2017-11-03.zip` come from?

Reply:
I have uploaded to our repo. Because Intel Edison cannot use git submodule, clone the azure c sdk is through bash is very hard to maintain and update, so I upload a latest release source code of c sdk.